ftplugin/markdown.vim

```diff
@@ -717,7 +717,7 @@ function! s:MarkdownHighlightSources(force)
             else
                 let include = '@' . toupper(filetype)
             endif
-            let command = 'syntax region %s matchgroup=%s start="^\s*```\s*%s$" matchgroup=%s end="\s*```$" keepend contains=%s%s'
+            let command = 'syntax region %s matchgroup=%s start="^\s*```\s*%s.*$" matchgroup=%s end="\s*```$" keepend contains=%s%s'
```

Comment:
Per [this comment](https://github.com/preservim/vim-markdown/pull/362#issuecomment-408423480) this regex is too aggressive. Do you have an alternative that gets the job done without introducing unintended matches?